paperqa/utils.py

```diff
@@ -419,3 +418,13 @@ async def _get_with_retrying(
 
 def union_collections_to_ordered_list(collections: Iterable) -> list:
     return sorted(reduce(lambda x, y: set(x) | set(y), collections))
+
+
+def pqa_directory(name: str) -> str:
```

Comment:
Are you open to returning `Path` instead of `str`? It's much easier to work with `pathlib`, especially in context of opening files or extending the path